pynest/examples/EI_clustered_network/network_params.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# sim_params.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""PyNEST EI-clustered network: Network Parameters
+------------------------------------------------
+
+A dictionary with parameters defining the network and neuron parameters.
+
+"""
+
+import numpy as np
+
+net_dict = {
+    ############################################
+    # neuron parameters
+    ############################################
+    # neuron model
+    "neuron_type": "iaf_psc_exp",
+    # Resting potential [mV]
+    "E_L": 0.0,
+    # Membrane capacitance [pF]
+    "C_m": 1.0,
+    # Membrane time constant for excitatory neurons [ms]
+    "tau_E": 20.0,
+    # Membrane time constant for inhibitory neurons [ms]
+    "tau_I": 10.0,
+    # Refractory period [ms]
+    "t_ref": 5.0,
+    # Threshold for excitatory neurons [mV]
+    "V_th_E": 20.0,
+    # Threshold for inhibitory neurons [mV]
+    "V_th_I": 20.0,
+    # Reset potential [mV]
+    "V_r": 0.0,
+    # synaptic time constant for excitatory synapses [ms]
+    "tau_syn_ex": 5.0,
+    # synaptic time constant for inhibitory synapses [ms]
+    "tau_syn_in": 5.0,
+    # synaptic delay [ms]
+    "delay": 0.1,
+    # Feed forward excitatory input [rheobase current]
+    "I_th_E": 1.25,
+    # Feed forward inhibitory input [rheobase current]
+    "I_th_I": 0.78,
+    # distribution of feed forward input,
+    # I_th*[1-delta_I_../2, 1+delta_I_../2]
+    "delta_I_xE": 0.0,  # excitatory
+    "delta_I_xI": 0.0,  # inhibitory
+    # initial membrane potential
+    "V_m": "rand",
```

Comment:
Can you provide a little more detail here, e.g. 

```suggestion
    # initial membrane potential: either a float (in mV) to initialize all neurons to a fixed value or "rand" for randomized values
    "V_m": "rand",
```

Where are the parameters (mean, sigma) for the random initialization?